### DISCUSSION TITLE
Likely outside of scope feature suggestion: Bulk editing of properties

### DISCUSSION THREAD

**Discussion by JunglePredator** [pdfarranger/pdfarranger#973] [Ideas]
Putting this in discussions as I believe it'll be outside the scope of the project. Think it's a an idea worth mentioning however.

Suggesting the ability to load multiple PDFs into PDF Arranger at one time & then, by an 'open files' ribbon, be able to select which PDFs are having some universal properties or other features edited. This would also allow people to move back & forth for the PDF in focus.

The example is if one makes or edits multiple PDFs, to be able to select all the PDFs & set the or a new author, with out having to open & type in the data multiple times.

**Comment by kbengs** [comment]
Could this be done with a drop down menu in the properties dialog, from where we could select from which pdf to read the properties/metadata ?
